pkg/i18n/i18n.go

```diff
@@ -17,7 +17,7 @@ type Localizer struct {
 
 func NewTranslationSetFromConfig(log *logrus.Entry, configLanguage string) (*TranslationSet, error) {
 	if configLanguage == "auto" {
-		language := detectLanguage(jibber_jabber.DetectLanguage)
+		language := detectLanguage(jibber_jabber.DetectIETF)
```

Comment:
Yes, it will return `en-AU`, and in function `NewTranslationSet()` ([link](https://github.com/jesseduffield/lazygit/blob/master/pkg/i18n/i18n.go#L39)), there is a boolean test to tell the key of `GetTranslationSets` is contained in the `language` string. Since `en-AU` has the prefix of `en`, the strings of `en` will be merged.

I've tried the following condition:
1. only keep following keys in `TranslationSet`: `"pl"`, `"en"`, `"zh"`, `"ja"`, `"ko"` (no `"zh-CN"` and `"zh-TW"`
2. use `detectIETF` to get language-region code, its `"zh-TW"`
3. run the `lazygit`, it will use the localization strings of `"zh"`

This is similar to the condition that `"en-AU"` use the localization strings of `"en"`.

Reply:
Ah, good to know. Thanks for doing that deep dive!